osu.Game/Screens/Menu/IntroWelcome.cs

```diff
@@ -0,0 +1,152 @@
+{
+    public class IntroWelcome : IntroScreen
+    {
+        protected override string BeatmapHash => "64E00D7022195959BFA3109D09C2E2276C8F12F486B91FCF6175583E973B48F2";
+        protected override string BeatmapFile => "welcome.osz";
+        private const double delay_step_two = 2142;
+        private SampleChannel welcome;
+
+        [BackgroundDependencyLoader]
+        private void load(AudioManager audio)
+        {
+            Seeya = audio.Samples.Get(@"Intro/welcome/seeya");
+
+            if (MenuVoice.Value)
+                welcome = audio.Samples.Get(@"Intro/welcome/welcome");
+        }
+
+        protected override void LogoArriving(OsuLogo logo, bool resuming)
+        {
+            base.LogoArriving(logo, resuming);
+
+            if (!resuming)
+            {
+                welcome?.Play();
+                StartTrack();
+                Scheduler.AddDelayed(delegate
+                {
+                    PrepareMenuLoad();
+
+                    logo.ScaleTo(1);
+                    logo.FadeIn();
+
+                    Scheduler.Add(LoadMenu);
+                }, delay_step_two);
+
+                LoadComponentAsync(new WelcomeIntroSequence
+                {
+                    RelativeSizeAxes = Axes.Both
+                }, AddInternal);
+            }
+        }
+
+        public override void OnSuspending(IScreen next)
+        {
+            this.FadeOut(300);
+            base.OnSuspending(next);
+        }
+
+        private class WelcomeIntroSequence : Container
+        {
+            private Sprite welcomeText;
+            private LogoVisualisation visualizer;
+            private Container elementContainer;
+            private Container circleContainer;
+            private Circle blackCircle;
+
+            [BackgroundDependencyLoader]
+            private void load(TextureStore textures)
+            {
+                Origin = Anchor.Centre;
+                Anchor = Anchor.Centre;
+                Children = new Drawable[]
+                {
+                    new Container
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        AutoSizeAxes = Axes.Both,
+                        Children = new Drawable[]
+                        {
+                            elementContainer = new Container
+                            {
+                                AutoSizeAxes = Axes.Both,
+                                Children = new Drawable[]
+                                {
+                                    visualizer = new LogoVisualisation
+                                    {
+                                        RelativeSizeAxes = Axes.Both,
+                                        Anchor = Anchor.Centre,
+                                        Origin = Anchor.Centre,
+                                        Alpha = 0.5f,
+                                        AccentColour = Color4.Blue,
```

Comment:
This does nothing at the moment. The accent color gets overriden in `LogoVisualisation`